src/test/java/timetables_sort_use_case/application_business/TestTimetablesSortInputBoundary.java

```diff
@@ -0,0 +1,20 @@
+package timetables_sort_use_case.application_business;
+
+public class TestTimetablesSortInputBoundary implements TimetablesSortInputBoundary {
```

Comment:
Is this also a mocked up input boundary for tests? If so, consider using anonymous classes.

Reply:
same problem as mentioned in testalltimetablesview